src/components/ReturnCallsCard.vue

```diff
@@ -1,58 +1,66 @@
 <template>
   <div>
-    <b-card>
-      <b-tabs pills card>
-        <!--- Current Call --->
-        <b-tab title="Current Call" class="current-call-tab" active>
-          <CurrentCallCard />
-          <!--- See All Button --->
-          <div id="See-All-btn">
-            <base-button class="text-crisis-cleanup-yellow-900 align-right">
-              See All
-            </base-button>
-          </div>
-        </b-tab>
-        <!---Scripts Tab--->
-        <b-tab title="Scripts" class="scripts-tab" active>
-          Scripts
-          <!--- See All Button --->
-          <div id="See-All-btn">
-            <base-button class="text-crisis-cleanup-yellow-900 align-right">
-              See All
-            </base-button>
-          </div>
-        </b-tab>
-      </b-tabs>
-    </b-card>
+    <base-button
+      variant="outline"
+      class="m-3 px-5 py-2"
+      :action="
+        () => {
+          isShowingModal = true;
+        }
```

Comment:
No need to wrap this in `{ }`. 

Some common arrow functions:

**Returning a single value/method** :

```js
const myFunction = () => true; // the 'return' is implicit here
console.log(myFunction()) // outputs: true

const anotherFunction = () => someMethod();
const roundUp = (num1, num2) => Math.ceil(num1*num2);
console.log(roundUp(1.2, 1.7)) // outputs: 3
```

**With side effects**

```js
const myFunction = () => {
   console.log('im a side effect');
   someOtherFunction();
    return true // note the explicit return
};


```

**Returning an object**

```js

const dynamicObject = (arg1) => ({
    keyOne: arg1,
});

console.log(dynamicObject(10)) // outputs: { keyOne: 10 }
```

**Returning a (singular) assignment**

```js

// generally won't be using this one as a const, but as an anonymous function 
const assignSomething = (state) => (someVar = state);

```

Reply:
That's how it was on storybook, hence why it's wrapped like that, however, I've removed them.